flutter/lib/src/default_integrations.dart

```diff
@@ -444,6 +410,30 @@ class _LoadAndroidImageListIntegrationEventProcessor extends EventProcessor {
 
     return event;
   }
+
+  static bool needsSymbolication(SentryEvent event) {
```

Comment:
This method could be e.g. an extension method of `SentryEvent` so you could easily do `event.needsSymbolication()`